input/docs/extending/modules/best-practices.md

```diff
@@ -0,0 +1,200 @@
+> modules built against older versions might not be compatible with future versions of Cake (this is especially true when a future version of Cake introduces breaking changes).
+>
+> Cake versions which are API compatible with a specific module will be shown when the module is added to the Cake website.
+
+----------------------------------------------------------------------------------------------------
+
+**_§2.2_** **Do** reference a newer version than Cake `1.0.0` if the module requires a specific functionality.
+
+> **Why?** If a specific feature of Cake is required in a module the lowest version of Cake which introduces this feature should be referenced
+> to have access to the feature and the best support for different versions of Cake.
+
+----------------------------------------------------------------------------------------------------
+
+**_§2.3_** **Do** update module to a newer version of Cake if a version of Cake with breaking changes becomes available.
+
+> **Why?** Cake will output a warning that the module was built against an incompatible version of Cake and the module might no longer work.
+> It is incumbent on module authors to upgrade their references and publish new version of their NuGet packages
+
+----------------------------------------------------------------------------------------------------
+
+**_§2.4_** **Avoid** dependencies to `Cake.Core` and `Cake.Common` in the NuGet package.
+
+> **Why?** Those references are being implicitly added by the Cake engine.
+
+<blockquote class="blockquote">
+  <p>
+    <strong>Example:</strong>
+  </p>
+  <ul class="nav nav-tabs">
+      <li class="active"><a data-toggle="tab" href="#dotnet">dotnet CLI / MSBuild</a></li>
+      <li><a data-toggle="tab" href="#nuget">nuget.exe CLI</a></li>
+  </ul>
+
+  <div class="tab-content">
+      <div id="dotnet" class="tab-pane fade in active">
+         <p>
+              References to <code>Cake.Core</code> and <code>Cake.Common</code> need to be marked as private assets in the project file:
+          </p>
+          <p>
+<pre><code class="language-xml hljs">&lt;ItemGroup&gt;
+  &lt;PackageReference Include="Cake.Core" Version="1.0.0" PrivateAssets="All" /&gt;
+  &lt;PackageReference Include="Cake.Common" Version="1.0.0" PrivateAssets="All" /&gt;
+&lt;/ItemGroup&gt;
+</code></pre>
+          </p>
+      </div>
+      <div id="nuget" class="tab-pane fade">
+          <p>
+              When using a <code>nuspec</code> file omit the references to <code>Cake.Core</code> and <code>Cake.Common</code>
+          </p>
+      </div>
+  </div>
+</blockquote>
+
+## .NET target version
+
+**_§2.5_** **Do** target `netstandard2.0`.
+
+> **Why?** Targeting to `netstandard2.0` should work for most modules to support all available Cake runners, operating systems and platforms.
+> Depending on the module it might be required to target additional platforms.
+
+**_§2.6_** **Do not** include additional targets such as `net461` and `net5.0`.
```

Comment:
Style vocabulary defines `Do`, `Consider` and `Avoid` sentences. Therefore we should use here an Avoid sentence:

```suggestion
**_§2.6_** **Avoid** targeting additional frameworks such as `net461` or `net5.0`.
```

Reply:
:white_check_mark: Fixed. Thanks!